doc/ale.txt

```diff
@@ -319,6 +329,12 @@ ALEGetLinters(filetype)                                       *ALEGetLinters()*
   Return all of linters configured for a given filetype as a |List| of
   |Dictionary| values in the format specified by |ALEAddLinter()|.
 
+ALEGetStatusLine()                                           *ALEGetStatusLine()*
+  Return a formatted string that can be added to the statusline.
+	The output's format is defined in |`g:ale_statusline_format`|.
+  To enable it, the following should be present in your |statusline| settings: >
+  %{ALEGetStatusLine()}
+
```

Comment:
Not necessary, the code block is stopped when no more indentation (2 spaces or more)